### PR TITLE
Config with c code

### LINKED ISSUE #42: Add support to release Linux AArch64 wheels
Problem
---------
On aarch64, ‘pip install zope.container’ build the wheels from source code and then installs it. It requires the user to have a development environment installed on his system. Also, it takes more time to build the wheels than downloading and extracting the wheels from PyPI.

Resolution
-----------
On aarch64, ‘pip install zope.container’ should download the wheels from PyPI.

@icemac, Please let me know your interest in releasing aarch64 wheels. I can help with this.

### PR DESCRIPTION
**Pull request by icemac**
* Lint the code.
* Add support for Python 3.9 and 3.10.

Fixes #42.

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -1,13 +1,19 @@
-[nosetests]
-nocapture=1
-cover-package=zope.container
-cover-erase=1
-with-doctest=0
-where=src
+# Generated from:
+# https://github.com/zopefoundation/meta/tree/master/config/c-code
+[bdist_wheel]
+universal = 1
```
Comment (sallner) [inline setup.cfg:4]: Is universal wheel reasonable here? I thought it actually contains c-code.

File: src/zope/container/contained.py [hunk 2]
```diff
@@ -871,7 +882,7 @@ class DecoratorSpecificationDescriptor(
         zope.interface.declarations.ObjectSpecificationDescriptor):
     """Support for interface declarations on decorators
 
-    >>> from zope.interface import *
+    >>> from zope.interface import Interface, directlyProvides, implementer
```
Comment (sallner) [inline src/zope/container/contained.py:885]: 👍🏾

File: src/zope/container/_compat.py [hunk 3]
```diff
@@ -0,0 +1,6 @@
+try:
+    # PY2
+    text_type = unicode
+except NameError:
+    # PY3
+    text_type = str
```
Comment (sallner) [inline src/zope/container/_compat.py:6]: Is this actually used more than once?

**Review by sallner — APPROVED** [review]
Looks good in general, but can't say anything about building wheels stack.

File: setup.cfg (hunk 1 above)
Comment (icemac) [reply setup.cfg:4]: Yes it is reasonable, we do this for all repositories containing C code.

File: src/zope/container/_compat.py (hunk 3 above)
Comment (icemac) [reply src/zope/container/_compat.py:6]: I am not sure. But it is in the _compat module, so it will go away sooner or later.

**Comment by icemac** [comment]
Thank you for reviewing this PR. 😃

**Pull request merged**